components/collector/tests/source_collectors/azure_devops/test_source_up_to_dateness.py

```diff
@@ -37,7 +37,7 @@ async def test_age_of_file(self):
         self.assert_measurement(
             response,
             value=self.expected_age,
-            landing_url=f"{self.url}/_git/repo?path=README.md&version=GBmaster",
+            landing_url=f"{self.url}/_git/repo?path=README.md&version=GBmain",
```

Comment:
Does this come from the change in `shared_data_model.parameters.Branch`; is this the only instance of that class?

Reply:
Yes and yes :-)

Azure DevOps only uses the branch parameter for the source-up-to-dateness metric. There are other branch parameters used by Azure DevOps, namely branches to ignore and target branches to include, but they are lists.

Reply:
So, can the second part of #8045 be achieved?

Reply:
Oops. Forgot about that part. I'll have a look!

Reply:
I don't think it's worth. See analysis in #8045. Agree?

Reply:
Yep makes sense! At best we could make `SonarQube` use `Branch`.. in a different story ;) Let's merge this?